src/style.css

```diff
@@ -1 +1,16 @@
 /* styles go here */
+html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.registerForm fieldset {
+  margin: 20px 0;
+}
+
+.registerForm fieldset legend {
+  font-weight: bold;
+}
```

Comment:
Remember, any other styles besides margins, should be browser default.

Reply:
![image](https://user-images.githubusercontent.com/98764641/225877659-7d554c92-5c6e-4e3e-9fd9-1ae09943d502.png)